facia/public/amp_remote.html

```diff
@@ -74,7 +74,7 @@
         }
 
         function attachKruxSegments(config) {
-            if (localStorage.getItem('kxsegs') !== null) {
+            if (localStorage && localStorage.getItem('kxsegs') !== null) {
```

Comment:
[Here](https://github.com/guardian/frontend/pull/12983#discussion_r64717316) is a thread about local storage's wild behaviour.